pkg/cmd/reset.go

```diff
@@ -100,5 +105,40 @@ func runReset(opts *resetOpts) error {
 
 	s.Logger.Warnln("this command will require an explicit confirmation starting with the next minor release (v1.3)")
 
+	fmt.Println("The following nodes will be reset. The Kubernetes cluster running on those nodes will be permanently destroyed ")
+	for _, node := range s.Cluster.ControlPlane.Hosts {
+		fmt.Printf("\t- reset control plane node %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+	for _, node := range s.Cluster.StaticWorkers.Hosts {
+		fmt.Printf("\t- reset static worker nodes %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+
+	if s.DynamicClient == nil {
+		err = kubeconfig.BuildKubernetesClientset(s)
+		if err != nil {
+			s.Logger.Warnln("Failed to build Kubernetes client set.")
```

Comment:
```suggestion
			return err
```

Reply:
Seem like there was previous discussion about this exact line and change. So please ignore mine message.